controllers/certificaterequest_controller.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"context"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	certmanv1alpha1 "github.com/openshift/certman-operator/api/v1alpha1"
+)
+
+// CertificateRequestReconciler reconciles a CertificateRequest object
+type CertificateRequestReconciler struct {
+	client.Client
+	Scheme *runtime.Scheme
+}
+
+//+kubebuilder:rbac:groups=certman.managed.openshift.io,resources=certificaterequests,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=certman.managed.openshift.io,resources=certificaterequests/status,verbs=get;update;patch
+//+kubebuilder:rbac:groups=certman.managed.openshift.io,resources=certificaterequests/finalizers,verbs=update
+
+// Reconcile is part of the main kubernetes reconciliation loop which aims to
+// move the current state of the cluster closer to the desired state.
+// TODO(user): Modify the Reconcile function to compare the state specified by
+// the CertificateRequest object against the actual cluster state, and then
+// perform operations to make the cluster state reflect the state specified by
+// the user.
+//
+// For more details, check Reconcile and its Result here:
+// - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.11.2/pkg/reconcile
+func (r *CertificateRequestReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	_ = log.FromContext(ctx)
+
+	// TODO(user): your logic here
+
+	return ctrl.Result{}, nil
+}
```

Comment:
this doesn't look right to me :laughing:

Reply:
Oh I missed removing this! Thanks to catch it :+1: 

**Background**: What happened here is that when new controller is initialised using `operator-sdk` command, the new file is directly created under `./controllers` directory in the new version. However, as part of the migration I wanted to maintain directory structure so had manually copied controller directories from `./pkg/controller/` in old version. Missed removing this file later on. For the actual controller can refer to this [diff](https://github.com/openshift/certman-operator/pull/225/files#diff-0285b65fbbf63a5eb3c040c52cfac34e69791b73cfc5e3adeb59954b6432a7ad).

Reply:
that makes sense, thanks for clarifying!